R/postprocess.R

```diff
@@ -403,3 +382,77 @@ enw_quantiles_to_long <- function(posterior) {
   long[, quantile := as.numeric(quantile) / 100]
   return(long[])
 }
+
+#' Build the ord_obs `data.table`.
+#'
+#' @param obs Observations as pulled from nowcast$latest[[1]].
+#' @param max_delay Whole number representing the maximum delay
+#' in units of the timestep.
+#' @param internal_timestep The internal timestep in days.
+#' @param timestep The timestep to be used. This can be a string
+#' ("day", "week", "month") or a numeric whole number representing
+#' the number of days.
+#' @param sample String, "get_sample" or "no_sample".
+#' @param nowcast If getting posterior samples, the fit to pull
```

Comment:
I think this is actually a dataframe with a .draw column vs a fit (I think its the output of cmdstanr draws?)